bin/server_docker/ubuntu/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+MAINTAINER Valdas Petrulis <petrulis.valdas@gmail.com>
```

Comment:
Personal email intended?:) Just in case you missed it.

Reply:
Yes, I dont care